packages/paypal-express-integration/src/paypal-express-script-loader.ts

```diff
@@ -0,0 +1,31 @@
+import { LoadScriptOptions, ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { PaypalHostWindow, PaypalSDK } from './paypal-sdk';
+
+export default class PaypalScriptLoader {
+    private _window: PaypalHostWindow;
+
+    constructor(private _scriptLoader: ScriptLoader) {
+        this._window = window;
+    }
+
+    async loadPaypal(merchantId = ''): Promise<PaypalSDK> {
+        const scriptSrc = '//www.paypalobjects.com/api/checkout.min.js';
+        const options: LoadScriptOptions = {
+            async: true,
+            attributes: { 'data-merchant-id': merchantId },
+        };
+
+        merchantId
+            ? await this._scriptLoader.loadScript(scriptSrc, options)
+            : await this._scriptLoader.loadScript(scriptSrc);
+
+        if (!this._window.paypal) {
+            throw new PaymentMethodClientUnavailableError();
+        }
+
+        return this._window.paypal;
+    }
```

Comment:
We can try to update this method like this:

```suggestion
    async loadPaypal(merchantId = ''): Promise<PaypalSDK> {
        const scriptSrc = '//www.paypalobjects.com/api/checkout.min.js';
        const options: LoadScriptOptions = {
            async: true,
            ...(merchantId && {
                attributes: { 'data-merchant-id': merchantId },
            })
        };

        await this._scriptLoader.loadScript(scriptSrc, options);

        if (!this._window.paypal) {
            throw new PaymentMethodClientUnavailableError();
        }

        return this._window.paypal;
    }
```

Reply:
fixed